pyproject.toml

```diff
@@ -16,7 +16,7 @@ classifiers = [
 description = "Interface to control systems via GraphQL over websockets"
 dependencies = [
     "typing-extensions;python_version<'3.8'",
-    "strawberry-graphql",
```

Comment:
I'd suggest sticking a comment here about the reason for the pin, and when it can be removed.

Reply:
I think given strawberry volatility being still 0.x we always want to pin and keep it regularly updated after having tested - i.e. don't plan to unpin entirely at least until they reach 1.0. But comment could explain reason for this particular version.

Reply:
Yes good idea. I've added a comment about this specific issue but also added a recommendation that we continue to specify a version as the Strawberry module seems fairly volatile.